_faq/mentor_obligations.md

```diff
@@ -10,3 +14,5 @@ Participate in weekly meetings with your student.
 * Review your students pull requests timely.
 * Get in contact with the admins if any even remotely
 potential problems arise.
+
+If you are interested in mentoring, please contact one of the admins indicating the reason your OSS work is suitable to be included as a project to be mentored under the coala banner. List of admins is [here](http://projects.coala.io/#/mentors).
```

Comment:
"OSS work" -> "open source project"

"banner" -> "administrative umbrella".

new paragraph for "List of .."